[RHACS] ROX-19745: Updating build time network policy tools documentation (#64889)
openshift/openshift-docs: modules/roxctl-netpol-connectivity-map-command-options.adoc

// Module included in the following assemblies:
//
// * operating/build-time-network-policy-tools.adoc

:_content-type: REFERENCE
[id="roxctl-netpol-connectivity-map-command-options_{context}"]
= roxctl netpol connectivity map command options

The `roxctl netpol connectivity map` command supports the following options:

[%autowidth,options="header"]
|===

|Option |Description

|`--fail`
|Fail on the first encountered error. The default value is `false`.

|`--focus-workload string`
|Focus on connections of a specified workload name in the output.

|`-h`, `--help`
|View the help text for the `roxctl netpol connectivity map` command.

|`-f`, `--output-file string`
|Save the connections list output into a specific file.

|`-o`, `--output-format string`

> |Configure the connections list in a specific format. The supported formats are `txt`, `json`, `md`, `dot`, and `csv`. The default value is `txt`.

Reviewer: ```suggestion
|Configure the output format. The supported formats are `txt`, `json`, `md`, `dot`, and `csv`. The default value is `txt`.
```
